test/artefacts/calculate-agricultural-holiday-entitlement/different-number-of-days.txt

```diff
@@ -0,0 +1,12 @@
+What date will your holiday start?
+
+
+
+This is the first day that you will take off work as leave
+
+
+
+
+
+
+
```

Comment:
It wasn't intentional. I only included things that were already being rendered in the templates. However, it feels as if we should be rendering the `date_question` options. I'm going to investigate how hard that will be.